test/user.js

```diff
@@ -323,11 +380,15 @@ describe('user', () => {
     });
   });
   describe('setup', () => {
+    const email = 'bob@example.com';
```

Comment:
As with the 'invalidEmail' variable, this also doesn't change. Its scope is the 'setup' describe. I don't really mind moving it but I'd like to understand the reasons behind it :confused: